completion/available/vagrant.completion.bash

```diff
@@ -54,96 +54,111 @@ _vagrant() {
 	prev="${COMP_WORDS[COMP_CWORD - 1]}"
 	commands="box cloud destroy global-status halt help hostmanager init login package plugin port powershell provision push rdp reload resume scp snapshot ssh ssh-config status suspend up upload validate vbguest version winrm winrm-config"
 
-	if [ $COMP_CWORD == 1 ]; then
-		COMPREPLY=($(compgen -W "${commands}" -- ${cur}))
+	if [ "$COMP_CWORD" == 1 ]; then
```

Comment:
I'd rather suggest

```suggestion
	if ((COMP_CWORD == 1)); then
```

The `==` operator in `[` is a Bash extension. POSIX only defines a single `=`. Then one could consider Bash's conditional command, `[[ $COMP_CWORD == 1 ]]`, but then the arithmetic command `((COMP_COWRD == 1))` is more natural in this context.

The same applies to all the occurrences of later `[ "$COMP_CWORD" == N ]`.

Reply:
will do...